erpnext/selling/page/point_of_sale/point_of_sale.py

```diff
@@ -14,8 +14,21 @@ def get_items(start, page_length, price_list, item_group, search_value="", pos_p
 	serial_no = ""
 	batch_no = ""
 	barcode = ""
-
+	warehouse = ""
+	display_items_in_stock = 0
 	item_code = search_value
+	
+	if pos_profile:
+		pos_result = frappe.db.sql("""select 
```

Comment:
Use ORM instead of SQL where possible

```
warehouse, display_items_in_stock = frappe.db.get_value('POS Profile', pos_profile, ['warehouse', 'display_items_in_stock'])
```